tests/reference_jobfile_hdd

```diff
@@ -1,4 +1,6 @@
 timeout 5s
 hdd 1
-hdd-bytes 100m
-hdd-write-size 4m
+hdd-bytes 100M
+hdd-opts direct, fsync, wr-rnd
```

Comment:
Do we need to be concerned with possibly malformed lists (trailing comma, leading comma, zero values, illegal values)?  That is, are these things which we expect Arca to address, or does it just pass them through to Stress-ng?

Reply:
This is a sample stress-ng input file that is used to compare against the input that is generated by the schema. The schema handles converting the list input to the comma-separated list that the stress-ng input file uses. The schema validation should handle what's needed here.